src/transformers/models/bridgetower/configuration_bridgetower.py

```diff
@@ -0,0 +1,403 @@
+        width (`int`, *optional*, defaults to 768):
+            Dimensionality of the encoder layers and the pooler layer.
+        layers (`int`, *optional*, defaults to 12):
+            Number of hidden layers in vit model.
+        vit_layernorm_init_from_vit (`bool`, *optional*, defaults to `False`):
+            Whether to init vit LayerNorm from vit.
+        vit_layernorm_shared (`bool`, *optional*, defaults to `True`):
+            Whether vit's LayerNorm layers are shared.
+        patch_size (`int`, *optional*, defaults to 16):
+            The size (resolution) of each patch in vit.
+        vit_remove_last (`bool`, *optional*, defaults to `False`):
+            Whether to remove vit's last layer.
+        transformer_width (`int`, *optional*, defaults to 512):
+            Width of vit's transformer.
+
+    Example:
+
+    ```python
+    >>> from transformers import BridgeTowerVisionConfig
+
+    >>> # Initializing a BridgeTower BridgeTower/bridgetower-base style configuration for the vision model
+    >>> configuration = BridgeTowerVisionConfig()
+
+    >>> # Accessing the configuration
+    >>> configuration
+    ```"""
+    model_type = "bridgetower_vision_model"
+
+    def __init__(
+        self,
+        embed_dim=512,
+        input_resolution=224,
+        width=768,
+        layers=12,
+        patch_size=16,
+        transformer_width=512,
+        output_resolution=288,
+        stop_gradient=False,
+        vit_layernorm_shared=True,
+        vit_remove_last=False,
+        vit_layernorm_init_from_vit=False,
+        **kwargs
+    ):
+        super().__init__(**kwargs)
+
+        self.embed_dim = embed_dim
+        self.input_resolution = input_resolution
+        self.width = width
+        self.layers = layers
+        self.patch_size = patch_size
+        self.transformer_width = transformer_width
+        self.output_resolution = output_resolution
+        self.stop_gradient = stop_gradient
+        self.vit_layernorm_shared = vit_layernorm_shared
+        self.vit_remove_last = vit_remove_last
+        self.vit_layernorm_init_from_vit = vit_layernorm_init_from_vit
+
+    @classmethod
+    def from_pretrained(cls, pretrained_model_name_or_path: Union[str, os.PathLike], **kwargs) -> "PretrainedConfig":
+        config_dict, kwargs = cls.get_config_dict(pretrained_model_name_or_path, **kwargs)
+
+        if config_dict.get("model_type") == "bridgetower":
+            config_dict = config_dict["text_config"]
+
+        if "model_type" in config_dict and hasattr(cls, "model_type") and config_dict["model_type"] != cls.model_type:
+            logger.warning(
+                f"You are using a model of type {config_dict['model_type']} to instantiate a model of type "
+                f"{cls.model_type}. This is not supported for all configurations of models and can yield errors."
+            )
+
+        return cls.from_dict(config_dict, **kwargs)
+
+
+class BridgeTowerTextConfig(PretrainedConfig):
```

Comment:
```suggestion
# Copied from transformers.models.roberta.configuration_roberta.RobertaConfig with RoBERTa->BridgeTower, roberta-base->BridgeTower/bridgetower-base
class BridgeTowerTextConfig(PretrainedConfig):
```
If everything is copied from RoBERTa here, you can add a Copied from statement above the class, run make fix-copies to make sure both are aligned

Reply:
Few configs differ from roberta, thus didn't add the "#Copied from.." statement

Reply:
Ok makes sense!